demisto_sdk/tests/integration_tests/create_id_set_integration_test.py

```diff
@@ -0,0 +1,141 @@
+import json
+
+from click.testing import CliRunner
+
+from demisto_sdk.__main__ import main
+from TestSuite.test_tools import ChangeCWD
+from TestSuite.utils import IsEqualFunctions
+
+CREATE_ID_SET_CMD = "create-id-set"
+
+
+class TestCreateIdSet:  # Use classes to speed up test - multi threaded py pytest
+    def test_create_id_set_with_excluded_items(self, mocker, repo):
+        """
+        Given
+            - Running create-id-set command
+
+        When
+            - some items should be excluded from the id set duo to mismatch in the marketplaces
+
+        Then
+            - Ensure create-id-set passes.
+            - Ensure no error occurs.
+            - Ensure items are being excluded from the id set
+        """
+        import demisto_sdk.commands.create_id_set.create_id_set as cis
+        import demisto_sdk.commands.find_dependencies.find_dependencies as find_dependencies
+        from demisto_sdk.tests.test_files.create_id_set import (
+            excluded_items_by_pack, excluded_items_by_type,
+            packs_dependencies_results)
+
+        pack = repo.create_pack('CreateIdSetPack')
+        integration = pack.create_integration('integration')
+        integration.create_default_integration()
+
+        with open('demisto_sdk/tests/test_files/create_id_set/unfiltered_id_set.json') as id_set_file:
+            mock_id_set = json.load(id_set_file)
+        mocker.patch.object(find_dependencies, "save_dict_of_sets")
```

Comment:
does it bother you if it runs this function?
if not, we can remove the mock so the test will be slimmer